### PR TITLE
Target release 4/25 - Android agent v7.3.1 release notes

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - Android agent v7.3.1 release notes

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-731.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+subject: Android agent
+releaseDate: '2024-04-23'
+version: 7.3.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.3.1.zip'
+---
+
+## Fix
+
+* Resolve a SHA-signed error during build time by disabling JAR signing verification allows the agent plugin to read the classes without issues.
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-731.mdx:10]: I'd remove `allows the agent plugin to read the classes without issues`.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-731.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* Resolve reported ANRs during event persistance process.
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-731.mdx:11]: Rather than _process_, how about `serialization`?

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Review by rhetoric101 — APPROVED** [review]
@ywang-nr thanks for adding these release notes!

**Comment by rhetoric101** [comment]
I have to do an administrative override because the Netlify checks didn't complete. I looked at the preview in Netlify.

**Pull request merged**